dist/s_clang-scan

```diff
@@ -50,9 +50,9 @@ grep 'No bugs found' $t > /dev/null && exit 0
 # clang-scan outputs errors/warnings with a leading file path followed by a
 # colon, followed by the string "error:" or "warning:".
 cat dist/s_clang-scan.diff |
-    egrep '^[a-z\./].*error: |^[a-z\./].*warning: ' | sed -e 's/:.*//' > $a
+    egrep '^[a-z\./].*error: |^[a-z\./].*warning: ' | sed -e 's/:.*:.*://' | sort > $a
```

Comment:
I'm wondering why this change is necessary.

Reply:
The reason we sorted the warning is that they were being generated in different order causing the clang-analyzer test to fail.